libs/journeys/ui/src/components/TextResponse/TextResponse.tsx

```diff
@@ -89,7 +89,8 @@ export const TextResponse = ({
       getTextResponseValues(
         { label, placeholder, hint },
         journey?.journeyCustomizationFields ?? [],
-        variant ?? 'default'
+        variant ?? 'default',
+        journey?.template ?? false
```

Comment:
```suggestion
        journey?.template 
```
see mu comment about making this value nullable, makes it easier for future devs to consume